File: saude_java/app/src/main/java/saude_java/RCQ.java
package saude_java;

public class RCQ {

    public static double calcular(double cintura, double quadril) {
        if (quadril <= 0) {
            throw new IllegalArgumentException("Quadril deve ser maior que zero");
        }
        double rcq = cintura / quadril;
        return Math.round(rcq * 100.0) / 100.0;  // arredonda para duas casas decimais
    }

    public static String classificacao(double rcq, String sexo) {
        if (sexo.equalsIgnoreCase("masculino")) {
            return rcq > 0.9 ? "Risco aumentado" : "Baixo risco";
        } else if (sexo.equalsIgnoreCase("feminino")) {
            return rcq > 0.85 ? "Risco aumentado" : "Baixo risco";
        } else {
            throw new IllegalArgumentException("Sexo deve ser 'masculino' ou 'feminino'");
        }
    }
}

File: saude_python/src/saude_python/rcq.py
class RCQ:
    @staticmethod
    def calcular(cintura, quadril):
        if quadril <= 0:
            raise ValueError("Quadril deve ser maior que zero")
        rcq = cintura / quadril
        return round(rcq, 2)

    @staticmethod
    def classificacao(rcq, sexo):
        if sexo == "masculino":
            return "Risco aumentado" if rcq > 0.9 else "Baixo risco"
        elif sexo == "feminino":
            return "Risco aumentado" if rcq > 0.85 else "Baixo risco"
        else:
            raise ValueError("Sexo deve ser 'masculino' ou 'feminino'")
